types/level-codec/index.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for level-codec 9.0
+// Project: https://github.com/Level/abstract-leveldown
```

Comment:
Is this an incorrect project? I couldn't found the interfaces on this d.ts

Reply:
yes that is the incorrect project, let me correct this :o